docs/entities/cookbook.md

```diff
@@ -12,11 +12,20 @@ cli.get_cookbooks()
 This will return a list of [cookbooks](/docs/commonly_used_data_types/cookbook.md), see link for more details on what that response looks like.
 
 ### Get Top Results
-This function will get you the top runs of the recipe group you input and is called like the following:
+This function will get you the top runs of the recipe group you input. This function can be called using either of the two APIs:
+
+#### Old API:
+
 ```
 cli.get_cookbook_top_results(recipe_group_id, limit)
 ```
 
+#### New API:
```

Comment:
Again what is this distinction if it's the same function?

Reply:
Explained the difference and added example code.